src/Provider/ApiServiceProvider.php

```diff
@@ -65,7 +65,7 @@ public function register()
             'Dingo\Api\Console\Command\Docs',
         ]);
 
-        if (class_exists('Illuminate\Foundation\Application', false)) {
+        if (class_exists('Illuminate\Contracts\Foundation\Application', false)) {
```

Comment:
You're correct, like I said I had a hard time following your poor explanation. I've reverted to the original line and would like to continue trying to get this merged in.